www/caddy/src/opnsense/scripts/OPNsense/Caddy/caddy_certs.php

```diff
@@ -27,61 +27,83 @@
  *    POSSIBILITY OF SUCH DAMAGE.
  */
 
+namespace OPNsense\Caddy;
+
 require_once "config.inc";
 
-use OPNsense\Core\Config;
-
-$configObj = Config::getInstance()->object();
-$temp_dir = '/var/db/caddy/data/caddy/certificates/temp/';
-
-// Traverse through certificates
-foreach ($configObj->cert as $cert) {
-    $cert_refid = (string)$cert->refid;
-    $cert_content = base64_decode((string)$cert->crt);
-    $key_content = base64_decode((string)$cert->prv);
-    $cert_chain = $cert_content;
-
-    // Handle CA and possible intermediate CA to create a certificate bundle
-    if (!empty($cert->caref)) {
-        foreach ($configObj->ca as $ca) {
-            if ((string)$cert->caref === (string)$ca->refid) {
-                $ca_content = base64_decode((string)$ca->crt);
-                $cert_chain .= "\n" . $ca_content;
-
-                if (!empty($ca->caref)) {
-                    foreach ($configObj->ca as $parent_ca) {
-                        if ((string)$ca->caref === (string)$parent_ca->refid) {
-                            $parent_ca_content = base64_decode((string)$parent_ca->crt);
-                            $cert_chain .= "\n" . $parent_ca_content;
-                            break;
-                        }
-                    }
-                }
+use OPNsense\Trust\Ca;
+use OPNsense\Trust\Cert;
+use OPNsense\Trust\Store as CertStore;
+
+$writeFileIfChanged = function ($filePath, $content) {
```

Comment:
```suggestion
function writeFileIfChanged($filePath, $content) {
```

Might be easier to read

Reply:
This here is a trick:

https://www.php.net/manual/en/functions.anonymous.php

It avoids having this error when `make style`

```
  1 | WARNING | [ ] A file should declare new symbols (classes, functions, constants, etc.) and cause no other side effects, or it should execute logic with side effects, but should not do both. The first symbol is defined on line
    |         |     37 and the first side effect is on line 30.
```